<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KiradaOlanlarFormu</class>
 <widget class="QWidget" name="KiradaOlanlarFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kirada Olan Araçlar</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="txt_arama_kira"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ARAMA YAP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>KİRALAMA LİSTESİ</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTableWidget" name="tbl_liste"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>710</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SİL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_teslim_et">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>710</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TESLİM ET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
